platform-emf/src/main/java/com/softicar/platform/emf/editor/EmfAttributeValueFrame.java

```diff
@@ -0,0 +1,116 @@
+package com.softicar.platform.emf.editor;
+
+import com.softicar.platform.common.core.interfaces.INullaryVoidFunction;
+import com.softicar.platform.emf.attribute.IEmfAttribute;
+import com.softicar.platform.emf.attribute.input.IEmfInput;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+import com.softicar.platform.emf.validation.result.IEmfDiagnostic;
+import com.softicar.platform.emf.validation.result.IEmfValidationResult;
+
+public class EmfAttributeValueFrame<R extends IEmfTableRow<R, ?>, V> extends AbstractEmfAttributeValueFrame<R, V> {
```

Comment:
I think we should add 1 or 2 sentences of Javadoc here, since this thing is quite important, and quite magic.

Reply:
added